app/views/trainees/employing_schools/_results.html.erb

```diff
@@ -13,7 +13,7 @@
 <%= f.govuk_radio_buttons_fieldset :employing_school_id, legend: nil do %>
   <%= f.hidden_field :school_value, value: "true" %>
   
-    <% @schools.each_with_index do |school, index| %>
+    <% @schools.to_h[:schools].each_with_index do |school, index| %>
```

Comment:
I would prefer this, (which would require us to make a name change in the controller as well)
```suggestion
    <% @school_search.schools.each_with_index do |school, index| %>
```